app/src/main/java/edu/wisc/ece/pinpoint/pages/newpin/NewPinFragment.java

```diff
@@ -0,0 +1,137 @@
+package edu.wisc.ece.pinpoint.pages.newpin;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.viewpager2.widget.ViewPager2;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ScrollView;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.Task;
+import com.google.android.material.tabs.TabLayout;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.firestore.DocumentReference;
+
+import edu.wisc.ece.pinpoint.R;
+import edu.wisc.ece.pinpoint.data.Pin;
+import edu.wisc.ece.pinpoint.data.Pin.PinType;
+import edu.wisc.ece.pinpoint.pages.profile.ProfileFragmentAdapter;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link NewPinFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ *
+ */
```

Comment:
Do we need this?

Reply:
Seems leftover from boilerplate, I'll remove it :+1: